docs/changes/newsfragments/6599.new

```diff
@@ -1 +1,2 @@
 Added a new feature to find links in a parameter chain by parameter type
+Added new methods to find instruments in a parameter chain by parameter type
```

Comment:
You should create a file called 6616.new otherwise this does not get linked to the correct pr in the changelog.

Reply:
Thanks for letting me know. I will create 6616.new